### ISSUE TITLE
Hardware issue: logo pressed

### ISSUE THREAD

**Issue by joshkordek3** [microsoft/pxt-microbit#3966]
**Describe the bug**
When you have a logo pressed block in your code and try to upload it to a microbit V1 then it will have an error where it doesn't do anything

**To Reproduce**
1. Go to 'makecode.microbit.org'
2. Click on 'create new project'
3. Write some code that includes 'on logo pressed'
4. Upload to a V1.5 microbit (or really any V1 microbit)
5. The microbit "crashes"

**Expected behavior**
it ignores the code in the "on logo pressed"

**micro:bit version (please complete the following information):**
V1.5

**Desktop (please complete the following information):**
Google Chrome: Version 89.0.4389.116 (Official Build)  (64-bit)
Platform: 13729.72.0 (Official Build) stable-channel gnawty
Channel: stable-channel
Firmware Version: Google_Gnawty.5216.239.156
ARC Enabled: false
ARC: 
Enterprise Enrolled: false
Developer Mode: false

**Comment by abchatra** [comment]
Hmm. on logo pressed is v2 only block and won't work on v1 as hardware doesn't support it. 

@joshkordek3 is this a request for better error message?

**Comment by abchatra** [comment]
@carlosperate

**Comment by joshkordek3** [comment]
@abchatra yes, it should just ignore the code, not try to execute it and show the 'anonymous error' thing on the screen, it should at least say WHY there is an error.

**Comment by abchatra** [comment]
Abhijith to repro and see if there is a better error message on the hardware

**Comment by abchatra** [comment]
Old bug closing. No plans to address.
